Dockerfile

```diff
@@ -328,5 +328,10 @@ WORKDIR ${AIRFLOW_HOME}
 
 ENV AIRFLOW__CORE__LOAD_EXAMPLES="false"
 
+ARG AIRFLOW_WEBSERVER_HOST_PORT=8080
```

Comment:
I do no think we need to have it configurable. Simply  8080 should be good enough?. That would make it much simpler.